src/coreComponents/constitutive/capillaryPressure/CapillaryPressureBase.hpp

```diff
@@ -92,10 +92,13 @@ class CapillaryPressureBase : public ConstitutiveBase
 
   struct PhaseType
   {
-    static constexpr integer OIL            = 0;
-    static constexpr integer GAS            = 1;
-    static constexpr integer WATER          = 2;
-    static constexpr integer MAX_NUM_PHASES = 3;
+    enum : integer
+    {
+      OIL            = 0,
+      GAS            = 1,
+      WATER          = 2,
+      MAX_NUM_PHASES = 3,
```

Comment:
Yeah. I just replaced `static constexpr` variables with unscoped enums because the latter don't require a definition in the cpp file (although I was the one who recommended using `static constexpr` in the first place...). Although we (currently) don't switch on these values, I agree that `MAX_NUM_PHASES` should perhaps be declared outside of enum.

Reply:
No we do not `switch` on `enums`, but maybe that could be a refactoring idea in the future if we manage to find some resources more dedicated to refactoring and pure code  🚀